site/src/template.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Title
```

Comment:
Should this file be called `atomics.template`, so that it isn't built into the site?

Reply:
Updated